packages/boiler-config-webpack/src/make-webpack-config.js

```diff
@@ -0,0 +1,208 @@
+import fs from 'fs';
+import assign from 'lodash/assign';
+import merge from 'lodash/merge';
+import isPlainObject from 'lodash/isPlainObject';
+import omit from 'lodash/omit';
+import {join} from 'path';
+import webpack from 'webpack';
+import formatter from 'eslint-friendly-formatter';
+import getLoaderPluginConfig from './get-loader-plugin-config';
+//import getExcludes from './gather-commonjs-modules';
+import createMultipleEntries from './make-multiple-entries';
+import applyAddons from './utils/apply-addons';
+
+export default function(config, defaultConfig, opts = {}) {
+  const {
+    isMainTask,
+    quick,
+    sources,
+    utils,
+    environment,
+    publicPath,
+    webpackConfig
+  } = config;
+  const {
+    entry,
+    srcDir,
+    scriptDir,
+    buildDir,
+    globalBundleName,
+    mainBundleName
+  } = sources;
+  const {
+    babel: babelParentConfig = {},
+    hashFunction,
+    hot,
+    expose,
+    multipleBundles,
+    paths,
+    vendors
+  } = webpackConfig;
+  const {jsBundleName} = paths;
+  const {isDev, isIE} = environment;
+  const {addbase} = utils;
+
+  const {
+    externals,
+    preLoaders,
+    loaders,
+    postLoaders,
+    plugins,
+    rules,
+    configFile
+  } = getLoaderPluginConfig(config, {dirs: opts.dirs});
+
+  const commons = {vendors};
+  const {context} = defaultConfig;
+
+
+  if (isMainTask) {
+    const devPlugins = [
+      new webpack.HotModuleReplacementPlugin()
+    ];
+
+    const hmrOpts = [
+      `path=${publicPath}__webpack_hmr`,
+      'reload=true'
+    ];
+    const hotEntry = [
+      `webpack-hot-middleware/client?${hmrOpts.join('&')}`
+    ];
+
+    /**
+     * If a shims file exists compile it with the `entry` or `vendors`
+     */
+    const shimFile = addbase(srcDir, scriptDir, 'shims.js');
+    const hasShims = fs.existsSync(shimFile);
+    let taskEntry, omitEntry, multiBase, glob, buildMultiBundles;
+
+    taskEntry = omit(entry, globalBundleName);
+
+    if (isPlainObject(multipleBundles)) {
+      ({omitEntry, base: multiBase, glob} = multipleBundles);
+      buildMultiBundles = true;
+    } else {
+      buildMultiBundles = multipleBundles;
+    }
+
+    /*eslint no-inner-declarations:0*/
+    function manageAdditions(mainBundle, method = 'unshift') {
+      const {omitPolyfill} = babelParentConfig;
+      const additions = Object.keys(expose);
+
+      if (!omitPolyfill && mainBundle.indexOf('babel-polyfill') === -1) {
+        additions.unshift('babel-polyfill');
+      }
+      //otherwise load the modules we want to expose
```

Comment:
extract adding the `babel-polyfill` in the entry to inside of the `config` function in `boiler-addon-webpack-loaders-babel`